### PR TITLE
FE-1281 - Remove use of `generateCypressEnv` script when running integration tests. Remove end-to-end tests.

### PR DESCRIPTION
**Pull request by nicklemmon**
[FE-1281](https://sparkpost.atlassian.net/browse/FE-1281)

### What Changed

- Removes end-to-end tests, adds placeholder `.gitkeep` file to preserve the directory for future use
- Removed contents of all `test-e2e` scripts - will be used in the future
- Removed references to `Cypress.env('USERNAME')` and `Cypress.env('PASSWORD')` - replaced with global constants that can (eventually) reference environmental variables as needed.

### How To Test

- Verify Cypress tests continue to pass
- Verify no end-to-end tests remain in the repo
- Verify the `generateCypressEnv` script was deleted

### To Do

- [ ] Incorporate feedback

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,14 +18,13 @@
...
     "test-ci": "CI=true npm test -- --coverage --colors --maxWorkers=2",
-    "test-cy-env": "node ./scripts/generateCypressEnv",
     "test-cy-run": "cypress run",
-    "test-e2e": "npm run test-cy-env && cypress open --config integrationFolder=cypress/tests/e2e",
-    "test-e2e-headless": "cypress run --config integrationFolder=cypress/tests/e2e",
-    "test-e2e-ci": "npm run start-app & wait-on http://localhost:3100 && npm run test-e2e-headless",
-    "test-integration": "npm run test-cy-env && TZ=America/New_York cypress open --config-file cypress.dev.json",
-    "test-integration-hibana": "npm run test-cy-env && TZ=America/New_York cypress open --config-file cypress.dev.json --env DEFAULT_TO_HIBANA=true",
-    "test-integration-headless": "npm run test-cy-env && TZ=America/New_York cypress run --config-file cypress.dev.json",
+    "test-e2e": "",
```
Comment (nstanard) [inline package.json:22]: might as well just remove these blank ones completely

File: cypress/tests/integration/domains/createPage.spec.js [hunk 2]
```diff
@@ -343,12 +343,17 @@ describe('The domains create page', () => {
...
       });
-      commonBeforeSteps();
-      cy.wait('@currentUserReq');
+      cy.stubRequest({
+        url: '/api/v1/account',
+        fixture: 'account/200.get.json',
+        requestAlias: 'accountDomainsReq',
+      });
+      cy.visit(PAGE_URL);
+      // cy.wait(['@currentUserReq', '@accountDomainsReq']);
```
Comment (nstanard) [inline cypress/tests/integration/domains/createPage.spec.js:356]: Maybe remove comment if it's not needed currently? Who knows when true e2e will come back. (Kinda wish we just hooked it up to work though. But integration tests do a really good job giving us confidence in our changes at least.)

**Review by nstanard — APPROVED** [review]
You might need to run a `git rm` command on the folder/file still in the repo/history.

image

couple small comments but nothing blocking

**Comment by nicklemmon** [comment]
> You might need to run a `git rm` command on the folder/file still in the repo/history.
> 
> image
> 
> couple small comments but nothing blocking

I actually want to preserve the folder because we will use it eventually - that's why I added that `.gitkeep` file in there.

**Comment by nicklemmon** [comment]
@nstanard Ready for another pass.

I ended up re-introducing those waits - that was actually a mistake on my end. I meant to re-introduce those after debugging some things.

**Pull request merged**